src/tests/data/PublicationFactory.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-underscore-dangle */
+import { ObjectId } from 'mongodb';
+import { PublicationModel } from '../../entities/Publication';
+
+class PublicationFactory {
+  public static async getRandomPublication() {
+    const pubs = await PublicationModel.aggregate().sample(1);
+    const pub = await PublicationModel.findById(new ObjectId(pubs[0]._id));
```

Comment:
Wouldn't `PublicationModel.aggregate().sample(1);` return the object as the first element in the array anyway? Why are we taking the ID and then looking up the object again?